backend/db-setup.js

```diff
@@ -5,7 +5,7 @@ const {
         port,
         auth: { user, pass }
     }
-} = require('./config');
+} = require('/src/config');
```

Comment:
also this whole file should be moved to /src/scripts

and we should run this script by "npm run dbsetup" so we should update package.json

Reply:
done